src/pages/auth/Signup.jsx

```diff
@@ -19,11 +19,7 @@ export function Signup() {
   const signUpHandler = () => {
     const { email, password, confirmPwd, firstName, lastName } = formData;
     if (
-      email &&
-      password &&
-      confirmPwd &&
-      firstName &&
-      lastName !== "" &&
+      (email && password && confirmPwd && firstName && lastName) !== " " &&
```

Comment:
have you tested it? This check will fail.

Reply:
`(email && password && confirmPwd && firstName && lastName)` will give you last truthy value.